courses/free-courses-en.md

```diff
@@ -682,6 +682,7 @@
 * [SoloLearn](https://www.sololearn.com/Course/Python/)
 * [The Python Tutorial](https://docs.python.org/3/tutorial/)
 * [Using Python for Research](https://www.edx.org/course/using-python-for-research) (edX Harvard)
+* [use python for machine learning](https://www.codewithharry.com/videos/ml-tutorials-in-hindi-10) (codewithharry)
```

Comment:
:warning: Following [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#titles) format guidelines:
> - No invented titles. We try to take titles from the resources themselves; contributors are admonished not to invent titles or use them editorially if this can be avoided. An exception is for older works; if they are primarily of historical interest, a year in parentheses appended to the title helps users know if they are of interest.

- Right resource title is: `Machine Learning Tutorials For Beginners Using Python In Hindi`
- Initial playlist video: https://www.codewithharry.com/videos/ml-tutorials-in-hindi-1

Please, use this as fix:

```markdown
* [Machine Learning Tutorials For Beginners Using Python In Hindi](https://www.codewithharry.com/videos/ml-tutorials-in-hindi-1) - CodeWithHarry
```

:information_source: Also I don't know if this is better than it related youtube playlist: https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG


:x: **Audio is in Hindi so move your commit** to `courses/free-courses-hi.md`: https://github.com/Akashgiri1020/free-programming-books/blob/patch-1/courses/free-courses-hi.md#Python

Reply:
### :warning: Moved to #5715, Duplicated at #5715

Reply:
Moved to #5728